exercises/book-store/.meta/src/reference/java/BookStore.java

```diff
@@ -4,47 +4,47 @@
 
 class BookStore {
 
-  private static final int BOOK_PRICE = 8, MAX_GROUP_SIZE = 5;
+    private static final int BOOK_PRICE = 8, MAX_GROUP_SIZE = 5;
 
-  private static double[] DISCOUNT_TIERS = {0, 5, 10, 20, 25};
+    private static double[] DISCOUNT_TIERS = {0, 5, 10, 20, 25};
 
-  double calculateBasketCost(final List<Integer> books) {
-    return calculateBasketCost(books, 0);
-  }
-
-  private double calculateBasketCost(final List<Integer> books, final double priceSoFar) {
-    if (books.size() == 0) {
-      return priceSoFar;
+    double calculateBasketCost(final List<Integer> books) {
+        return calculateBasketCost(books, 0);
     }
 
-    List<Integer> availableBookNumbers = books.stream()
-        .distinct()
-        .collect(Collectors.toList());
+    private double calculateBasketCost(final List<Integer> books, final double priceSoFar) {
+        if (books.size() == 0) {
+            return priceSoFar;
+        }
 
-    double minPrice = Double.MAX_VALUE;
+        List<Integer> availableBookNumbers = books.stream()
+            .distinct()
+            .collect(Collectors.toList());
 
-    for (int i = 0; i < availableBookNumbers.size(); i++) {
-      List<Integer> newGroupBooks = new ArrayList<>(availableBookNumbers.subList(0, i + 1));
-      List<Integer> remainingBooks = new ArrayList<>(books);
+        double minPrice = Double.MAX_VALUE;
 
-      for (final Integer newGroupBook : newGroupBooks) {
-        //noinspection UseBulkOperation - we want to remove _one_ of each book number, not _all_ of each book number.
-        remainingBooks.remove(newGroupBook);
-      }
+        for (int i = 0; i < availableBookNumbers.size(); i++) {
+            List<Integer> newGroupBooks = new ArrayList<>(availableBookNumbers.subList(0, i + 1));
+            List<Integer> remainingBooks = new ArrayList<>(books);
 
-      double price = calculateBasketCost(remainingBooks, priceSoFar + costOfGroupSize(newGroupBooks.size()));
-      minPrice = Math.min(minPrice, price);
-    }
+            for (final Integer newGroupBook : newGroupBooks) {
+            //noinspection UseBulkOperation - we want to remove _one_ of each book number, not _all_ of each book number.
+            remainingBooks.remove(newGroupBook);
```

Comment:
The indentation seems to have gone a bit wrong here for the two lines above? :)

Reply:
Oops! Fixed now :slightly_smiling_face: